test/cli/run.spec.ts

```diff
@@ -0,0 +1,388 @@
+      args: ["--extra-options-file", "____ERROR____FILE_DOES_NOT_EXIST"],
+      stdin: 'foo = "1"',
+    })).rejects.toThrow("Can't read from file");
+  });
+
+  it("handles formats", async() => {
+    await expect(exec({
+      args: ["--format"],
+    })).rejects.toThrow("option '--format <format>' argument missing");
+
+    await expect(exec({
+      args: ["--format", "BAD_FORMAT"],
+    })).rejects.toThrow("option '--format <format>' argument 'BAD_FORMAT' is invalid. Allowed choices are amd, commonjs, es, globals, umd.");
+  });
+
+  it("doesn't fail with optimize", async() => {
+    await expect(exec({
+      args: ["--optimize", "anything"],
+      stdin: 'foo = "1"',
+    })).resolves.toMatch(/deprecated/);
+
+    await expect(exec({
+      args: ["-O", "anything"],
+      stdin: 'foo = "1"',
+    })).resolves.toMatch(/deprecated/);
+
+    await expect(exec({
+      args: ["-O"],
+      stdin: 'foo = "1"',
+    })).rejects.toThrow("-O, --optimize <style>' argument missing");
+  });
+
+  it("outputs to a file", async() => {
+    const test_output = path.resolve(__dirname, "test_output.js");
+
+    expect(() => {
+      // Make sure the file isn't there before we start
+      fs.statSync(test_output);
+    }).toThrow();
+
+    await expect(exec({
+      args: ["-o", test_output],
+      stdin: "foo = '1'",
+    })).resolves.toBe("");
+    expect(fs.statSync(test_output)).toBeInstanceOf(fs.Stats);
+    fs.unlinkSync(test_output);
+
+    await expect(exec({
+      args: ["--output"],
+      stdin: "foo = '1'",
+    })).rejects.toThrow("-o, --output <file>' argument missing");
+
+    await expect(exec({
+      args: ["--output", "__DIRECTORY__/__DOES/NOT__/__EXIST__/none.js"],
+      stdin: "foo = '1'",
+    })).rejects.toThrow("Can't write to file \"__DIRECTORY__/__DOES/NOT__/__EXIST__/none.js\"");
+  });
+
+  it("handles plugins", async() => {
+    // Plugin, starting with "./"
+    const plugin = "./" + path.relative(
+      process.cwd(),
+      path.resolve(__dirname, "./fixtures/plugin.js")
+    );
+
+    await expect(exec({
+      args: [
+        "--plugin", plugin,
+        "--extra-options", '{"cli_test": {"words": ["foo"]}}',
+      ],
+      stdin: "var = foo:'1'",
+    })).rejects.toThrow("Label can't be a reserved word \"foo\"");
+
+    await expect(exec({
+      args: ["--plugin"],
+      stdin: "foo = '1'",
+    })).rejects.toThrow("--plugin <module>' argument missing");
+
+    await expect(exec({
+      args: ["--plugin", "ERROR BAD MODULE DOES NOT EXIST"],
+      stdin: "foo = '1'",
+    })).rejects.toThrow("Can't load module \"ERROR BAD MODULE DOES NOT EXIST\"");
+  });
+
+  it("handlers trace", async() => {
+    await expect(exec({
+      args: ["--trace"],
+      stdin: "foo = '1'",
+    })).resolves.toMatch("DefaultTracer: peg$DefaultTracer");
+  });
+
+  it("uses dash-dash", async() => {
+    await expect(exec({
+      args: ["--", "--trace"],
+    })).rejects.toThrow("no such file or directory, open '--trace'");
```

Comment:
Strange, but on Windows the error contains full path to the file, so changed to regexp
```suggestion
    // On Windows error contains the full path to the file: <work_dir>\--trace
    })).rejects.toThrow(/no such file or directory, open '[^']*--trace'/);
```